test/cppsuite/tests/run.cxx

```diff
@@ -127,7 +130,7 @@ main(int argc, char *argv[])
 {
     std::string cfg, config_name, test_name;
     int64_t error_code = 0;
-    const std::vector<std::string> all_tests = {"poc_test"};
+    const std::vector<std::string> all_tests = {"example_test", "poc_test"};
```

Comment:
Can we rename "example_test" to "obsolete_data_test"?

Reply:
"example_test" is a great name as a template so anyone can start from it. We could go for "template_test" if you'd like.